deploy/job-manager/capabilities_config.json

```diff
@@ -37,5 +37,12 @@
     "workflow-name",
     "comment"
   ],
-  "queryExtensions": []
+  "queryExtensions": [],
+  "authentication": {
+    "isRequired": true,
+    "scopes": [
```

Comment:
According to https://developers.google.com/+/web/api/rest/oauth, these 2 scopes below seems been deprecated, maybe it's better to use:
```
"openid",
"email",
"profile"
```

Reply:
I'm not sure about the effect of the `"openid"` here tho.